### PR TITLE
Demo title of PR

### PR DESCRIPTION
**Pull request by Bijay555**
fixes issue #65

### CODE REVIEW

File: src/data/contributors/bijay.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Bijay Adhikari",
+    "skills" : ["JS", "Python","Java","Sql"],
+    "cause" : "fun",
+    "image" : "Avatar2",
```
Comment (Abhishek-kumar09) [inline src/data/contributors/bijay.json:5]: That avatar is girl avatar, I think you mistakenly put it in your card:)
Though good cause to code. **#fun** :+1: 
image

Comment (Bijay555) [reply src/data/contributors/bijay.json:5]: can I change avatar?

**Comment by Bijay555** [comment]
I fixed avatar issue . .Is it okay??

**Comment by Abhishek-kumar09** [comment]
Happy to see you in @codeforcauseorg. Feel free to checkout other projects. Goodluck!

**Pull request merged**